src/applications/personalization/appointments/actions/index.js

```diff
@@ -179,15 +210,18 @@ export function fetchConfirmedFutureAppointments() {
 
         // Derive the facility.
         const facility =
-          facilitiesLookup[getStagingID(appointment.attributes?.facilityId)];
+          facilitiesLookup[getStagingID(appointment.attributes.facilityId)];
 
         accumulator.push({
           additionalInfo: getAdditionalInfo(appointment),
           facility,
-          id: appointment?.id,
-          isVideo: !isEmpty(appointment?.attributes?.vvsAppointments),
+          id: appointment.id,
+          isVideo: isVideoVisit(appointment.attributes),
           providerName: facility?.attributes?.name,
-          startsAt: startDate.toISOString(),
+          startsAt: getLocalAppointmentDate(
```

Comment:
This is the first major change. We are properly converting the VA appointment's date before storing it in `startsAt`.